Projects/views.py

```diff
@@ -354,3 +334,43 @@ def change_label(request):
         issue.save()
         return HttpResponse(res)
     else: return HttpResponseBadRequest()
+
+def new_issue(request):
+    print('adding issue')
+    if request.method == 'POST':
+        if request.user.is_staff:
+            print('in add issue method')
+            mentor = request.user
+            title_issue = request.POST.get('title_issue')
+            link_issue = request.POST.get('link_issue')
+            title_project = request.POST.get('title_project')
+            link_project = request.POST.get('link_project')
+            level = int(request.POST.get('level'))
+            points = int(request.POST.get('points'))
+
+            new_issue = Issues()
+            new_issue.mentor = mentor
+            new_issue.title_issue = title_issue
+            new_issue.link_issue = link_issue
+            new_issue.title_project = title_project
+            new_issue.link_project = link_project
+            new_issue.level = level
+            new_issue.points = points
+            new_issue.save()
+
+            issues=Issues.objects.all().order_by('-id')
+            paginator = Paginator(issues, 15)  # Show 15 issues per page
+            page = request.GET.get('page', 1)
+
+            try:
+                issues = paginator.get_page(page)
+            except PageNotAnInteger:
+                issues = paginator.get_page(1)
+                # issues = paginator.get_page(1)
+            except EmptyPage:
+                issues = paginator.get_page(paginator.num_pages)
+                issues = Issues.objects.none()
+            return render(request,'Projects/home.html',{'issues':issues})
+        else:
```

Comment:
can be improved :smile:

Reply:
snackbar is done well :+1: while new_issue view has lot to improve and error handling

Reply:
as I said in my previous PR of this issue this else part will never be executed because only staff user will see the add issue button on navbar so only they can add the issue. This was just for debugging

Reply:
Actually a non staff user or un authenticated users can perform this action using console :smile:

Reply:
that time else part is required